app/views/idv/doc_auth/agreement.html.erb

```diff
@@ -20,7 +20,7 @@
 <%= validated_form_for :doc_auth,
                        url: url_for,
                        method: 'put',
-                       html: { autocomplete: 'off', class: 'margin-top-2 js-consent-continue-form' } do |f| %>
+                       html: { autocomplete: 'off', class: 'margin-top-2 padding-bottom-1 js-consent-continue-form' } do |f| %>
```

Comment:
Similar, we could do `margin-bottom-5` and rely on collapsing margins.

```suggestion
                       html: { autocomplete: 'off', class: 'margin-top-2 margin-bottom-5 js-consent-continue-form' } do |f| %>
```

Doesn't make much a difference, though I think could be easier to think about the form having 2.5rem bottom margin, vs. a combination of its `0.5rem` bottom-padding and the next element's `2rem` top-margin.

Reply:
Mmmm, yeah, I get that. I should have been thinking about it as the large margin would set the distance. The margin collapsing thing is not something I think about normally.